src/Avalonia.Diagnostics/Diagnostics/Views/ConsoleView.xaml.cs

```diff
@@ -45,7 +45,9 @@ private void InputKeyDown(object sender, KeyEventArgs e)
             switch (e.Key)
             {
                 case Key.Enter:
+#pragma warning disable CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
                     vm.Execute();
+#pragma warning restore CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
```

Comment:
I think it's more common to suppress this warnings using "_" variable. What do you think?
```suggestion
                    _ = vm.Execute();
```